.github/workflows/master-to-relrc.yml

```
@@ -0,0 +1,25 @@
+name: Create rel/rc pull request
+on:
+  push:
+    branches:
+      - master
+jobs:
+  relrc-promotion:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: rel/rc
+      - name: Merge latest master to rel/rc
+        run: |
+          git fetch origin master:master
+          git reset --hard master
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: format('chore', ':', 'merge latest to rel/rc')
```

How come this needs `format()`? Also let's not confuse `master` with `latest`, which might refer to the `rel/latest` branch, which is something else.
```suggestion
          commit-message: 'chore: merge master to rel/rc'
```

---

format is needed because the `:` breaks the yml file.

---

even inside quotes? TIL

---

Removed per comments below.